src/pipelines/seas5_pipeline.py

```diff
@@ -101,7 +101,6 @@ def query_api(self, year, issued_month=None, fc_month=None):
 
     def process_data(self, raw_filename, year):
         raw_file_path = self.local_raw_dir / raw_filename
-        self.metadata["year_issued"] = year
```

Comment:
With this, I think we're missing the `['year_issued']` attribute getting added to the data if it's coming from before 2024